sos/report/plugins/openstack_nova.py

```diff
@@ -92,6 +92,8 @@ def setup(self):
         if self.get_option("all_logs"):
             self.add_copy_spec([
                 "/var/log/nova/",
+                "/var/log/{}*/nova*".format(self.apachepkg),
+                "/var/log/{}*/placement*".format(self.apachepkg),
```

Comment:
We collect the same `copy_spec` also in `openstack_placement` (same applies to `"/var/log/{}*/placement*.log".format(self.apachepkg),` few lines below). Two plugins can't collect the same `copy_spec` (I guess you know the reason).

Are we sure the two plugins wont be enabled concurrently any time? Or shouldnt we collect the `placement` logs just in `openstack_placement` plugin?

Reply:
On the Ubuntu platforms and how it is deployed, it is typically on 2 separate hosts. But, I brought these in from the RedHat section primarily.

But, it is technically possible of that happening where you have both the nova and placement on the same host.

I will re-evaluate this later